### ISSUE TITLE
控件无法访问，提示 Failed Resolution of:Landroid/view/accessibility/Accessibilitynodeinfoo$Extra Renderinginfo;

### ISSUE THREAD

**Issue by yangislet** [kkevsekk1/AutoX#727]
1. Autox.js 版本：629
2. Autox.js 下载渠道：https://github.com/kkevsekk1/AutoX/releases
3. Android 版本：Android 10
4. Android 机型：小米10
5. Android 系统类别：MIUI
6. VSCODE Autox.js 扩展版本（可选，如果是连接电脑的问题需要填）：
7. 问题描述：无法获取安卓view的AccessibilityNodeInfo
8. 执行脚本：  textContains("注册").findOnce(0).click();
9. 报错日志（可选）： 

> java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/accessibility/AccessibilityNodeInfo$ExtraRenderingInfo;
	at java.lang.reflect.Executable.getMethodReturnTypeInternal(Native Method)
	at java.lang.reflect.Method.getReturnType(Method.java:148)
	at org.mozilla.javascript.JavaMembers.extractGetMethod(JavaMembers.java:697)
	at org.mozilla.javascript.JavaMembers.findGetter(JavaMembers.java:684)
